src/ui/app_header/fair_header.h

```diff
@@ -109,18 +114,120 @@ void draw_header_bar(std::string_view title, ImFont *title_font, Style style) {
+    const auto utc = std::chrono::system_clock::to_time_t(clock);
     const auto len = strftime(utctime.data(), utctime.size(), "%H:%M:%S (UTC)", gmtime(&utc));
     TextRight(std::string_view(utctime.data(), len));
+    auto posBeneathClock = ImGui::GetCursorPos();
 
     // draw fair logo
     ImGui::SetCursorPos(topLeft);
-    const auto scale = titleSize.y / img_fair_h;
-    ImGui::Image((void *) (intptr_t) (style == Style::Light ? img_fair_tex : img_fair_tex_dark), ImVec2(scale * img_fair_w, scale * img_fair_h));
+    fair::RadialCircularMenu<1> leftMenu(localLogoSize, -20.f, 120.f);
+    ImGui::PushStyleColor(ImGuiCol_Button, ImVec4(0.0f, 0.0f, 0.0f, 0.0f));
+    ImGui::PushStyleColor(ImGuiCol_ButtonHovered, ImVec4(.8f, .8f, .8f, 0.4f));
+    ImGui::PushStyleColor(ImGuiCol_ButtonActive, ImVec4(0.0f, 0.0f, 0.0f, 0.0f));
+    ImGui::PushStyleVar(ImGuiStyleVar_FrameRounding, .5f * scale * logoSize.y);
+    const ImTextureID imgLogo = (void *) (intptr_t) (style == Style::Light ? imgFairLogo : imgFairLogoDark);
+    if (ImGui::ImageButton(imgLogo, localLogoSize) || ImGui::IsItemHovered()) {
+        using fair::RadialButton;
+        const bool wasAlreadyOpen = leftMenu.isOpen();
+        auto      &app            = DigitizerUi::App::instance();
+
+        ImGui::PushStyleColor(ImGuiCol_Button, { .3f, 1.0f, .3f, 1.f }); // green
+        leftMenu.addButton(
+                "\uF201", [&app](RadialButton &button) {
+                    app.mainViewMode = "View";
+                },
+                app.fontIconsSolidLarge, "switch to view mode");
+
+        leftMenu.addButton(
+                "\uF248", [&app](RadialButton &button) {
+                    app.mainViewMode = "Layout";
+                },
+                app.fontIconsSolidLarge, "switch to layout mode");
+        leftMenu.addButton(
+                "\uF542", [&app](RadialButton &button) {
+                    app.mainViewMode = "FlowGraph";
+                },
+                app.fontIconsSolidLarge, "click to edit flow-graph");
+        leftMenu.addButton(
+                "", [&app](RadialButton &button) {
+                    app.mainViewMode = "OpenSaveDashboard";
+                },
+                app.fontIconsSolidLarge, "click to open/save new dashboards");
+        ImGui::PopStyleColor();
+
+        if (wasAlreadyOpen && !ImGui::IsItemHovered()) {
+            fmt::print("was already open -> closing\n");
+            leftMenu.forceClose();
+        }
+    }
+
+    // right menu
+    auto                       &app = DigitizerUi::App::instance();
+    fair::RadialCircularMenu<2> rightMenu(localLogoSize, 75.f, 195.f);
+    ImGui::SetCursorPos(ImVec2(ImGui::GetIO().DisplaySize.x - localLogoSize.x, 0));
+    if (ImGui::Button("##rightMenuDummyButton", localLogoSize) || ImGui::IsItemHovered()) {
+        using fair::RadialButton;
+        using enum DigitizerUi::WindowMode;
+
+        ImGui::PushStyleColor(ImGuiCol_Button, { .3f, .3f, 1.0f, 1.f }); // blue
```

Comment:
The black text color on blue is not very visible.

Reply:
This requires some general UI-wide style-guide/handling... IMO out-of-scope for this PR. I can add another colour -- please suggest on -- until there is a more generic way in place.